js/modules/datetime.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- Hi, I would advise you to solve all the linter mistakes manually. Microverse does not encourage students to disable the Linters because it helps to catch bugs as well as to have clean and beautiful code.